chrome/edge-navigation.json

```diff
@@ -4,7 +4,7 @@
   "navItems": [
     {
       "title": "Inventory",
-      "filterable": false,
+      "filterable": true,
```

Comment:
You can remove the flag. It's true by default. I don't want to make an impression that people have to include the flag in order to appear in the dropdown.

Reply:
Good point!